x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/fingerprint.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiCode } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { FieldsConfig, from, to } from './shared';
+import { TargetField } from './common_fields/target_field';
```

Comment:
I don't think we can reuse the target field here. The help text indicates the input field is updated in place:
<img width="690" alt="Screen Shot 2021-05-25 at 9 14 57 AM" src="https://user-images.githubusercontent.com/40268737/119504288-d3417500-bd39-11eb-842e-6bb7786ec237.PNG">

However, the `target_field` actually defaults to `fingerprint` for this processor:
<img width="780" alt="Screen Shot 2021-05-25 at 9 16 31 AM" src="https://user-images.githubusercontent.com/40268737/119504389-e9e7cc00-bd39-11eb-8dc8-72cec20558f6.PNG">

In most cases, we expect multiple input fields so this makes more sense.

Reply:
Uhh nice catch! I've updated the helper text for the target field so that it says what it defaults to and removed the bits about updating it in place.

![Screenshot 2021-05-25 at 15 58 45](https://user-images.githubusercontent.com/1191206/119511182-76ae7c00-bd72-11eb-914c-135176744bc7.png)